grouper/fe/templates/permission.html

```diff
@@ -58,56 +58,30 @@
             <table class="table table-elist">
                 <thead>
                     <tr>
-                        <th class="col-sm-2">Group</th>
-                        <th class="col-sm-5">Argument</th>
+                        <th class="col-sm-2">Permission Grants</th>
+                        <th class="col-sm-5">Type</th>
                     </tr>
                 </thead>
                 <tbody>
-                {% for grant in group_grants %}
-                    <tr class="group-grant-row">
-                        <td class="grant-group">
-                            <a href="/groups/{{grant.group}}">{{grant.group}}</a>
-                        </td>
-                        <td class="grant-argument">
-                            {{grant.argument|default("(unargumented)", True)}}
-                        </td>
-                    </tr>
-                {% endfor %}
-                {% if not group_grants %}
-                    <tr class="no-group-grants">
-                        <td colspan="2" class="text-center"><em>This permission is not mapped to any
-                        groups. To grant this permission to a group, visit the group page.</em></td>
-                    </tr>
-                {% endif %}
-                </tbody>
-            </table>
 
-            <table class="table table-elist">
-                <thead>
-                    <tr>
-                        <th class="col-sm-2">Service Account</th>
-                        <th class="col-sm-5">Argument</th>
-                    </tr>
-                </thead>
-                <tbody>
-                {% for grant in service_account_grants %}
-                    <tr class="service-grant-row">
-                        <td class="grant-service">
-                            <a href="/users/{{grant.service_account}}">{{grant.service_account}}</a>
-                        </td>
-                        <td class="grant-argument">
-                            {{grant.argument|default("(unargumented)", True)}}
-                        </td>
-                    </tr>
-                {% endfor %}
-                {% if not service_account_grants %}
-                    <tr class="no-service-grants">
-                        <td colspan="2" class="text-center"><em>This permission is not mapped to
-                        any service accounts. To grant this permission to a service account, visit
-                        the service account page.</em></td>
-                    </tr>
-                {% endif %}
-                </tbody>
+                <tr class="permission-grant-row">
+                    <td class="grant-group">
+                        <a href="{{transfer_qs('/permissions/', permission.name, '/groups')}}">Permission Group Grants</a>
```

Comment:
How about `Grants to Groups` (and `Grants to Service Accounts`) instead? A little clearer IMO